crates/flox/src/commands/environment.rs

```diff
@@ -282,7 +288,9 @@ impl Activate {
         // We don't have access to the current PS1 (it's not exported), so we
         // can't modify it. Instead set FLOX_PROMPT_ENVIRONMENTS and let the
         // activation script set PS1 based on that.
-        let error = Command::new(activation_path.join("activate"))
+        let activation_script = activation_path.join("activate");
+        debug!("running activation script: {}", activation_script.display());
+        let error = Command::new(activation_script)
```

Comment:
question nonblocking: was committing this deliberate?

Reply:
Yes, I want the path to the activation script logged if we're in debug mode to help diagnose a `command not found` error.